rdr_service/model/study_nph_enums.py

```diff
@@ -4,9 +4,10 @@
 class StoredSampleStatus(messages.Enum):
     """Status of NPH StoredSample"""
 
-    RECEIVED = 0
-    SHIPPED = 1
-    DISPOSED = 2
+    UNSET = 0
+    RECEIVED = 1
+    SHIPPED = 2
+    DISPOSED = 3
```

Comment:
Can you also include a migration file to update the existing database to match the new values? For example, any existing stored samples that have a value of 1 should be updated to have a values of 2 to continue indicating that they're SHIPPED (rather than appearing as if they're RECEIVED).

Reply:
The plan was to backfill the `StoredSample.status` field value in `stable` database because it only has `0` values and `prod` has no rows in `StoredSample`

```
> SELECT ss.status, count(*) from nph.stored_sample ss GROUP BY ss.status;
status|count(*)|
------+--------+
     0|    3246|
```

Reply:
I like the idea of a migration file instead to do the same